### PR TITLE
Add dyslexia (illiteracy quirk) as a genetic mutation and disease symptom

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request

This is an expansion of #66648. The illiterate quirk is being added to genetics and virology as side effects named dyslexia.

## Why It's Good For The Game

More ways to gain and interact with the new quirk.

## Changelog

:cl:
add: Add dyslexia (illiteracy quirk) as a genetic mutation.
add: Add illiteracy as a effect for confusion disease symptom.
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
You might be shocked to hear that people with dyslexia can actually read and write

**Comment by timothymtorres** [comment]
> You might be shocked to hear that people with dyslexia can actually read and write

I have it and I can read and write fine however there are severe forms that can prevent people from being literate.

Not everyone that has Tourette's syndrome curses outloud.  Exaggerating disabilities for SS13 is silly but part of the memes.

**Comment by SkeletalElite** [comment]
Maybe make it a brain trauma instead of a genetic mutation and call it alexia

**Comment by tralezab** [comment]
Your pr does not do what it says it does.

**Comment by timothymtorres** [comment]
Dyslexia is already a brain trauma from the original illiteracy PR.  I'll be adding the disease symptom later tonight.

**Comment by ATH1909** [comment]
> > You might be shocked to hear that people with dyslexia can actually read and write
> 
> I have it and I can read and write fine however there are severe forms that can prevent people from being literate. Not everyone that has Tourette's syndrome curses outloud. Exaggerating disabilities for SS13 is silly but part of the memes.

maybe label it as "severe dyslexia" or "crippling dyslexia", then?

**Review by SuperNovaa41 — CHANGES_REQUESTED** [review]

File: code/datums/diseases/advance/symptoms/confusion.dm [hunk 1]
```diff
@@ -55,5 +60,13 @@
...
 	return
+
+/datum/symptom/confusion/on_stage_change(datum/disease/advance/A)
+	. = ..()
+	if(!.)
+		return FALSE
+	var/mob/living/carbon/M = A.affected_mob
+	if(A.stage >= 4 && illiterate)
+		ADD_TRAIT(M, TRAIT_ILLITERATE, DISEASE_TRAIT)
+	return TRUE
```
[the remark this answers is not in the document]
Comment (timothymtorres) [reply code/datums/diseases/advance/symptoms/confusion.dm]: Refactored with 047fa9004439664c634f84a8dd06895e6d0e1149

**Pull request merged**